app/backend/src/couchers/models.py

```diff
@@ -2756,3 +2757,15 @@ class AccountDeletionReason(Base):
     reason = Column(String, nullable=True)
 
     user = relationship("User")
+
+
+class RateLimitViolation(Base):
+    __tablename__ = "rate_limit_violations"
+
+    id = Column(BigInteger, primary_key=True)
+    created = Column(DateTime(timezone=True), nullable=False, server_default=func.now())
```

Comment:
`time`? More descriptive than `created`?

Reply:
To be consistent with the rest of the code base, I left it as `created` for now. Let me know if it bothers you a lot 😅